bundle/1.0.0-1/hive-operator.v1.0.0-1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,620 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    categories: OpenShift Optional
+    certified: 'false'
+    containerImage: quay.io/openshift-hive/hive:1.0.0
+    createdAt: '2020-03-23T22:36:00Z'
+    description: OpenShift cluster provisioning and management at scale.
+    support: Devan Goodwin
```

Comment:
Not entirely comfortable with this one :) but it's ok for now.

Reply:
I'm happy to change it -- what to?

Reply:
Nah it's ok for now.

Reply:
oops already changed it